Index: yuv2rgb.c
===================================================================
--- yuv2rgb.c	(revision 29417)
+++ yuv2rgb.c	(working copy)
@@ -168,26 +168,26 @@
 YUV2RGBFUNC(yuv2rgb_c_32, uint32_t, 0)
     LOADCHROMA(0);
     PUTRGB(dst_1,py_1,0,0);
-    PUTRGB(dst_2,py_2,0,1);
+    PUTRGB(dst_2,py_2,0,0);
 
     LOADCHROMA(1);
-    PUTRGB(dst_2,py_2,1,1);
+    PUTRGB(dst_2,py_2,1,0);
     PUTRGB(dst_1,py_1,1,0);
 
     LOADCHROMA(2);
     PUTRGB(dst_1,py_1,2,0);
-    PUTRGB(dst_2,py_2,2,1);
+    PUTRGB(dst_2,py_2,2,0);
 
     LOADCHROMA(3);
-    PUTRGB(dst_2,py_2,3,1);
+    PUTRGB(dst_2,py_2,3,0);
     PUTRGB(dst_1,py_1,3,0);
 ENDYUV2RGBLINE(8)
     LOADCHROMA(0);
     PUTRGB(dst_1,py_1,0,0);
-    PUTRGB(dst_2,py_2,0,1);
+    PUTRGB(dst_2,py_2,0,0);
 
     LOADCHROMA(1);
-    PUTRGB(dst_2,py_2,1,1);
+    PUTRGB(dst_2,py_2,1,0);
     PUTRGB(dst_1,py_1,1,0);
 ENDYUV2RGBFUNC()
 
