--- freetype-2.3.5.orig/include/freetype/config/ftoption.h	2008-09-07 13:00:59.000000000 +0200
+++ freetype-2.3.5/include/freetype/config/ftoption.h	2008-09-07 14:31:08.000000000 +0200
@@ -458,7 +458,7 @@
   /*   Do not #undef this macro here, since the build system might         */
   /*   define it for certain configurations only.                          */
   /*                                                                       */
-/* #define TT_CONFIG_OPTION_BYTECODE_INTERPRETER */
+#define TT_CONFIG_OPTION_BYTECODE_INTERPRETER
 
 
   /*************************************************************************/
@@ -505,7 +505,7 @@
   /*     ...                                                               */
   /*   }                                                                   */
   /*                                                                       */
-#define TT_CONFIG_OPTION_UNPATENTED_HINTING
+/* #define TT_CONFIG_OPTION_UNPATENTED_HINTING */
 
 
   /*************************************************************************/
