Index: control.c
===================================================================
--- control.c	(revision 324)
+++ control.c	(revision 325)
@@ -237,8 +237,8 @@
   for (i = 0; i < num_quads; i++)
   	value[i] = htonl (value[i]);
 
-  length = 4;
-  retval = (*d)->Write (d, (*d)->GetDevice (d), &full_addr, &value, &length,
+  length = 4 * num_quads;
+  retval = (*d)->Write (d, (*d)->GetDevice (d), &full_addr, value, &length,
       true, craw->generation);
   if (retval != 0) {
     fprintf (stderr, "Error writing (%x)...\n", retval);
